### PR TITLE
Remove require utils

### PR DESCRIPTION
**Pull request by rslibed**
Removing require utils due to deprecation

### CODE REVIEW

File: Share/Share/ShareViewModel.ts [hunk 1]
```diff
@@ -240,36 +243,40 @@ class ShareViewModel extends declared(Accessor) {
...
     const { isWGS84, isWebMercator } = point.spatialReference;
     // If spatial reference is WGS84 or Web Mercator, use longitude/latitude values to generate the share URL parameters
     if (isWGS84 || isWebMercator) {
       return promiseUtils.resolve(point);
     }
-    this._projectionPromise = requireUtils.when(moduleRequire, [
-      "esri/geometry/projection",
-      "esri/geometry/SpatialReference"
-    ]);
+    this._projectionPromise = projection.load();
```
Comment (driskull) [inline Share/Share/ShareViewModel.ts]: Can you use async/await here?

**Review by driskull — APPROVED** [review]
👍

File: Share/Share/ShareViewModel.ts (hunk 1 above)
Comment (rslibed) [reply Share/Share/ShareViewModel.ts]: Updated VM to use `async/await`

**Review by driskull — APPROVED** [review]
nice!!! 👍

**Pull request merged**